brainreg_segment/regions/layers.py

```diff
@@ -74,6 +74,13 @@ def add_existing_label_layers(
     return label_layer
 
 
+def add_region_from_existing_layer(selected_layer, label_layers):
+    """
+    Adds an existing label layer (e.g. from another plugin) to the list
+    """
+    label_layers.append(selected_layer)
```

Comment:
I think it might be worth having a check that the layer is a labels type here, and erroring if not? That way if this is used programattically it will error correctly. Then in `regions.py` where the gui stuff happens a `try ..., except ...` pattern can be used to catch the error and pop up a info box.

Reply:
Good idea. Added in 52d3d934d779c1375d43e55ba387fd416f84a31d.